container/qe-metrics

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+poetry run qe-metrics $@
```

Comment:
What does @ refer to?

Reply:
@oharan2 this passes any arguments to this command. This file creates a binary for we-metrics so we don't have to use "poetry run qe-metrics --arg1 --arg2..."

Reply:
It's not required, it's for convenience. But I'll remove it